clickhouse_connect/dbapi/cursor.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, client: Client):
         self.names = []
         self.types = []
         self._rowcount = 0
+        self._summary: List[Dict[str, str]] = []
```

Comment:
It's been a while since I looked at this, but can the cursor be reused?  Is that why you want to keep a list of the summaries instead of just the latest?

Reply:
Actually it is because of executemany